types/type-specs.tsx

```diff
@@ -168,7 +168,7 @@ if (params) {
 }
 
 const [] = useLocation({base: '/app'}); // $ExpectError
```

Comment:
Can you also fix this line as well? It was expecting an error on unknown options, but since `base` now is a valid name we could rename it to:
```suggestion
const [] = useLocation({foo: '/app'}); // $ExpectError
```